iothub/service/src/Common/Exceptions/ErrorCode.cs

```diff
@@ -66,7 +66,11 @@ public enum ErrorCode
         DeviceNotFound = 404001,
 
         JobNotFound = 404002,
-        PartitionNotFound = 404003,
+
+        /// <summary>
+        /// The error is internal to IoT Hub and is likely transient. For more details, see <see href="https://docs.microsoft.com/azure/iot-hub/iot-hub-troubleshoot-error-503003-partitionnotfound">503003 PartitionNotFound</see>.
+        /// </summary>
+        PartitionNotFound = 503003, // We do not handle this error code in our SDK
```

Comment:
I don't see that the service returns this either. Should we add these attributes too then?

```csharp
        [Obsolete("This error does not appear to be thrown by the service.")]
        [EditorBrowsable(EditorBrowsableState.Never)]
```